File: faktorial.py
print("Program Faktorial")
bilangan = int(input("Masukkan bilangan bulat (0-9): "))

if 0 <= bilangan <= 9:
    faktorial = 1
    for i in range(1, bilangan + 1):
        faktorial *= i
    print("Faktorial dari", bilangan, "adalah", faktorial)
else:
    print("Bilangan tidak valid. Silakan masukkan bilangan bulat antara 0-9.")

File: week2/faktorial.dart
void main() {
  int n = 5; // Ubah input di sini

  print("Faktorial dari ${n}: ${faktorial(n)}");
}

int faktorial(int n) {
  if (n == 0) {
    return 1;
  }
  return n * faktorial(n - 1);
}